src/provider/model/FinEnvironment.ts

```diff
@@ -111,16 +111,16 @@ export class FinEnvironment extends AsyncInit implements Environment {
             interface OFManifest {
                 shortcut?: {name?: string; icon: string};
                 // eslint-disable-next-line camelcase
-                startup_app: {uuid: string; name?: string; icon?: string};
+                startup_app?: {uuid: string; name?: string; icon?: string};
             }
 
             const application = fin.Application.wrapSync(identity);
             const applicationInfo = await application.getInfo();
 
-            const {shortcut, startup_app: startupApp} = applicationInfo.manifest as OFManifest;
+            const {shortcut, startup_app: startupApp} = (applicationInfo.manifest || {}) as OFManifest;
```

Comment:
Would still comment that we don't trust the `ApplicationInfo` interface that `manifest` will be non-null. Seems like the kind of thing where someone would otherwise look at the type and decide `||` check isn't necessary